docs/ledger/migrations/v5.0-migration.md

```diff
@@ -0,0 +1,105 @@
+# Migration Guide v5.0
```

Comment:
Missing messages and queries for basket:
- MsgUpdateBasketFee, MsgUpdateBasketFeeResponse
- MsgUpdateCurator, MsgUpdateCuratorResponse
- QueryBasketFeeRequest, QueryBasketFeeResponse

Reply:
Also the event:

- EventUpdateCurator

Reply:
Also `MsgTake` now includes `retirement_reason`.